docs/getting-started/first-steps-with-celery.rst

```diff
@@ -426,14 +425,14 @@ Worker doesn't start: Permission Error
     If you provide any of the :option:`--pidfile <celery worker --pidfile>`,
     :option:`--logfile <celery worker --logfile>` or
     :option:`--statedb <celery worker --statedb>` arguments, then you must
-    make sure that they point to a file/directory that's writable and
+    make sure that they point to a file or directory that's writable and
     readable by the user starting the worker.
 
 Result backend doesn't work or tasks are always in ``PENDING`` state
 --------------------------------------------------------------------
 
 All tasks are :state:`PENDING` by default, so the state would've been
-better named "unknown". Celery doesn't update any state when a task
+better named "unknown." Celery doesn't update the state when a task
```

Comment:
I believe the first change in this line should be reverted: `"unknown".` seems the correct version.